instrumentation-tests/src/main/assets/set_route_event_cleared.json

```diff
@@ -0,0 +1 @@
+{"events":[{"type":"setRoute","route":"{}","route_index":0,"leg_index":0,"event_timestamp":1599676809.101939,"result":{"route_state":"initialized","location":{"lat":38.56299513844654,"lon":-121.46685775094203,"time":159723.708431,"monotonicTimestampNanoseconds":159723708431000,"speed":0.0,"bearing":198.68202209472657,"altitude":-20.899999618530275,"accuracyHorizontal":10.494000434875489,"provider":"fused"},"route_index":0,"leg_index":0,"remaining_leg_distance":821.7999877929688,"remaining_leg_duration":157.041954097,"step_index":0,"remaining_step_distance":93.49351501464844,"remaining_step_duration":66.622300299,"state_message":"","in_tunnel":false,"predicted":0.0,"shape_index":0,"intersection_index":1,"road_name":"","off_road_proba":0.0,"key_points":[{"lat":38.56299513844654,"lon":-121.46685775094203,"time":159722.706546,"monotonicTimestampNanoseconds":159722706546000,"bearing":198.68202209472657}],"map_matcher_output":{"is_teleport":false,"matches":[]}},"delta_us":1932}],"version":"19.0.0","history_version":"1.0.0"}
```

Comment:
These files were updated here https://github.com/mapbox/mapbox-navigation-android/pull/4526
And then deleted here https://github.com/mapbox/mapbox-navigation-android/pull/4601

Reviving them in the MapboxHistoryTest because they require nav-native in order to read them.